packages/manager/apps/hycu/src/pages/edit-pack/EditPack.page.tsx

```diff
@@ -0,0 +1,243 @@
+                  disabled={product.planCode === currentPack || undefined}
+                  key={product.planCode}
+                  label={product.invoiceName}
+                  pricing={getRenewPrice(product.pricings)}
+                  subsidiary={subsidiary}
+                  onClick={() => {
+                    setSelectedPack(product.planCode);
+                  }}
+                ></OrderTile>
+              ))}
+            </OrderTileWrapper>
+            <div className="flex flex-row">
+              <OsdsButton
+                className="mr-4"
+                color={ODS_THEME_COLOR_INTENT.primary}
+                onClick={() => {
+                  navigate(
+                    urls.dashboard.replace(subRoutes.serviceName, serviceName),
+                  );
+                }}
+                slot="actions"
+                variant={ODS_BUTTON_VARIANT.ghost}
+              >
+                {tCommon('hycu_cta_cancel')}
+              </OsdsButton>
+              <OsdsButton
+                color={ODS_THEME_COLOR_INTENT.primary}
+                disabled={(!selectedPack && !orderLink) || undefined}
+                onClick={() => {
+                  setIsOrderInitiated(true);
+                  redirectToOrder();
+                }}
+                slot="actions"
+              >
+                {tCommon('hycu_cta_order')}
+              </OsdsButton>
+            </div>
+          </>
+        ) : (
+          <>
+            <OsdsTile className="mb-8">
+              <span slot="start">
+                <div className="flex flex-col gap-6 mb-6">
+                  <OsdsText
+                    level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+                    size={ODS_THEME_TYPOGRAPHY_SIZE._600}
+                    color={ODS_THEME_COLOR_INTENT.text}
+                  >
+                    {t('hycu_edit_pack_initiated_title')}
+                  </OsdsText>
+                  <OsdsText
+                    level={ODS_THEME_TYPOGRAPHY_LEVEL.subheading}
+                    size={ODS_THEME_TYPOGRAPHY_SIZE._800}
+                    color={ODS_THEME_COLOR_INTENT.text}
+                  >
+                    {t('hycu_edit_pack_initiated_description')}
+                  </OsdsText>
+                  <OsdsLink
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    target={OdsHTMLAnchorElementTarget._blank}
+                    referrerpolicy={
+                      ODS_LINK_REFERRER_POLICY.strictOriginWhenCrossOrigin
+                    }
+                    href={orderLink}
+                  >
+                    {orderLink}
+                    <span slot="end">
+                      <OsdsIcon
+                        className="ml-4 cursor-pointer"
+                        name={ODS_ICON_NAME.EXTERNAL_LINK}
+                        size={ODS_ICON_SIZE.xs}
+                        hoverable
+                      ></OsdsIcon>
```

Comment:
For information he exist a component to wrap it : https://github.com/ovh/manager/blob/develop/packages/manager-react-components/src/components/typography/links/links.component.tsx